client/client_test.go

```diff
@@ -2774,15 +2747,6 @@ func rootRoleCertID(t *testing.T, repo *NotaryRepository) string {
 	return rootKeys[0]
 }
 
-func verifyOnlyTrustedCertificate(t *testing.T, repo *NotaryRepository, certID string) {
-	certs, err := repo.CertStore.GetCertificatesByCN(repo.gun)
-	require.NoError(t, err)
-	require.Len(t, certs, 1)
-	id, err := trustmanager.FingerprintCert(certs[0])
-	require.NoError(t, err)
-	require.Equal(t, certID, id)
-}
-
 func TestRotateRootKey(t *testing.T) {
 	ts := fullTestServer(t)
 	defer ts.Close()
```

Comment:
Sorry, it's collapsed so I can't comment on the line, but there is a [very long comment](https://github.com/docker/notary/blob/remove-certstore/client/client_test.go#L2792) in this test that was added by #648 about `NotaryRepository.Update`'s handling of certs, which could probably be deleted?